packages/components/src/font-size-picker/next.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * External dependencies
+ */
+import { FontSizeControl } from '@wp-g2/components';
+
+/**
+ * Internal dependencies
+ */
+import { __unstableWithNext as withNext } from '../__next/context';
+
+export function withNextComponent( current ) {
+	return withNext( current, FontSizeControl, 'WPComponentsFontSizePicker' );
```

Comment:
It needs to be wrapped with:

`if ( process.env.COMPONENT_SYSTEM_PHASE === 1 ) {`

An alternative I tried is to pass a string name and keep the mapping inside `withNext`. It works but then you need to usually update code in two places.